src/components/CreateProposal/CreateProposalContainer.tsx

```diff
@@ -83,8 +101,11 @@ class CreateProposalContainer extends React.Component<IProps, IState> {
 
   handleSubmit = (event : any) => {
     event.preventDefault();
-    this.setState({ buttonEnabled: false });
-    this.props.createProposal(this.state.avatarAddress, this.state.title, this.state.description, this.state.nativeTokenReward, this.state.reputationReward, this.state.beneficiary);
+
+    if(!Object.keys(this.state.errors).length){
```

Comment:
more whitespace in general please, like between if and ( and between ) and {

Reply:
Should we add a linter and settle on specific linting rules?

Reply:
yes we should, there's a ticket for that. but not this second